projecttalentonboardingtask.client/src/components/Customer/Create.jsx

```diff
@@ -40,28 +52,32 @@ export default function Create({ isCreated }) {
             if (!response.ok) {
                 throw new Error('Network response was not ok');
             }
-            // Handle success           
-            console.log('Customer created successfully');
-
-            setMessage(<Message positive>
-                <p>The customer has been created successfully.</p>
-            </Message>)
-
-
+      
+            setMessage({ text: 'The customer has been created successfully.', type: 'positive' });
+            
             setTimeout(() => {
                 handleDisplayModal();
                 isCreated(true)
             }, 3000);
 
 
         } catch (error) {
-            setMessage(<Message negative>
-                <p>There was an error while creating the customer.</p>
-            </Message>)
+            setMessage({ text: 'There was an error while creating the customer.', type: 'negative' });
+            
             console.error('There was a problem creating customer:', error.message);
         }
     };
 
+    const renderMessage = () => {
+        if (!message.text) return null;
```

Comment:
Can you use a proper conditioning format? This is fine, but just for the sake of formality and for you to be aware as well, because you might or other devs might get confused as well. We can use something like
**if (!message.text {...}**